src/iris/vendors/iris_hipchat.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+
+import ujson
+import logging
+import requests
+import time
+from iris.constants import IM_SUPPORT
+
+logger = logging.getLogger(__name__)
+
+
+class iris_hipchat(object):
+    supports = frozenset(['hipchat'])
+
+    def __init__(self, config):
+        self.config = config
+        self.modes = {
+            'hipchat': self.send_message
+        }
+        self.proxy = None
+        if 'proxy' in self.config:
+            host = self.config['proxy']['host']
+            port = self.config['proxy']['port']
+            self.proxy = {'http': 'http://%s:%s' % (host, port),
+                          'https': 'https://%s:%s' % (host, port)}
+        self.token = self.config.get('auth_token')
+        self.room_id = self.config.get('room_id')
+        self.debug = self.config.get('debug')
+        self.endpoint_url = self.config.get('base_url')
+
+        self.params = {'auth_token': self.token}
+        self.notification_url = '{0}/v2/room/{1}/notification'.format(self.endpoint_url, self.room_id)
+        self.headers = {
+            'Content-type': 'application/json',
+            }
```

Comment:
slight nitpick: can you put the closing brace on the same level of indention as self.headers?